extensions/ql-vscode/src/cli.ts

```diff
@@ -115,6 +111,11 @@ interface BqrsDecodeOptions {
  */
 export class CodeQLCliServer implements Disposable {
 
+  /**
+   * CLI version where --format=DIL was introduced
+   */
+  private static DIL_VERSION = new SemVer('2.3.0');
```

Comment:
Nit: This made me think of the optimiser version and confused me. Perhaps something more boring and explicit, like `CLI_VERSION_WITH_DECOMPILE_KIND_DIL`? Also update comment to mention `--kind` instead of `--format`.

Reply:
That's a mouthful, but is fine.